partition_tables_templates/kickstart_dynamic.erb

```diff
@@ -0,0 +1,44 @@
+<%#
+kind: ptable
+name: Kickstart dynamic
+model: Ptable
+oses:
+- CentOS
+- Fedora
+- RedHat
+%>
+<%#
+
+When a partition template contains line starting with "#Dynamic", the contents
+will be put into %pre shell scriplet rather and it's expected it creates a file
+"/tmp/diskpart.cfg" which is then included into the Kickstart partitioning
+section. This allows generating parititioning scheme on the managed node befure
+Anaconda starts installation.
+
+This template is an example of such dynamic partitioning scheme. It follows
+recommended scheme of Red Hat Enterprise Linux for servers (no extra allocation
+for hibernation):
+
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/ch-swapspace
+
+-%>
+#Dynamic (do not remove this line)
+
+MEMORY=$((`grep MemTotal: /proc/meminfo | sed 's/^MemTotal: *//'|sed 's/ .*//'` / 1024))
+if [ "$MEMORY" -lt 2048 ]; then
+    SWAP_MEMORY=$(($MEMORY * 2))
+elif [ "$MEMORY" -lt 8192 ]; then
+    SWAP_MEMORY=$MEMORY
+elif [ "$MEMORY" -lt 65536 ]; then
+    SWAP_MEMORY=$(($MEMORY / 2))
+else
+    SWAP_MEMORY=32768
+fi
+
+cat <<EOF > /tmp/diskpart.cfg
+zerombr yes
```

Comment:
This failed for me. zerombr doesnot seem to have any sub commands\options.

Reply:
Yeah another overlook, thanks.